ADA/git_databricks.qmd

```diff
@@ -127,28 +120,34 @@ You can then click Save at the bottom of the page, and now your connection betwe
 
 ### Connecting to repos and cloning
 
+---
+
+Just like any other way that you've worked with git before, the first step is going to be to clone your repo inside Databricks. Git folders (previously called Repos in Databricks) can now be created in your Home folder. 
+
+In the blue menu on the left, click Workspace, and click the Home folder. The menu should look like this:
 
-Just like any other way that you've worked with git before, the first step is going to be to clone your repo inside Databricks. 
+![](../images/databricks-new-workspace.png)
 
-In the blue menu on the left, click Workspace, and then in the Workspace menu that appears, click Repos. The menus should look like this:
+Click the blue "Create" button in the top right corner, and then click "Git folder":
 
-![](../images/databricks-workspace-menu.png)
+![](../images/add-git-folder.png)
 
-On the Repos screen, click the grey "Add repo" button in the top right corner. The "Add Repo" window will appear:
 
-![](../images/databricks-add-repo.png)
```

Comment:
Should we be deleting the old files and reusing the name where possible? Would save us keeping all the old images in the latest repo too, they'll always be there in the Git history

Reply:
Yeah I agree with this - will delete unused ones now and will reuse names going forward